pkg/volume/glusterfs/glusterfs.go

```diff
@@ -1077,10 +1076,10 @@ func (plugin *glusterfsPlugin) ExpandVolumeDevice(spec *volume.Spec, newSize res
 
 	// Find out delta size
 	expansionSize := (newSize.Value() - oldSize.Value())
-	expansionSizeGB := int(volume.RoundUpSize(expansionSize, 1000*1000*1000))
+	expansionSizeGiB := int(volume.RoundUpSize(expansionSize, volume.GIB))
```

Comment:
Oh, one thing that just struck me: Since you introduce `RoundUpToGiB()`, why don't you use it?

Reply:
Because, we have to calculate difference and then use `RoundUpToGib` and `RoundUpToGiB` takes `resource.Quantity` as arguments..